lib/tasks/organization.rake

```diff
@@ -0,0 +1,7 @@
+task add_organization_id_to_users: :environment do
```

Comment:
@RushilBasappa Merge these two rake tasks into one.
You'll have to use `find_or_create` by `email_domain` to find the `organisation_id` and set it to `users` and `track_templates`.